tests/test_units.py

```diff
@@ -69,7 +77,7 @@ def test_get_unit(self):
 
         # Get CellML unit
         store = UnitStore()
-        assert str(store.get_unit('liter')) == 'liter'
+        assert str(store.get_unit('liter')).endswith('liter')
         assert isinstance(store.get_unit('ampere'), store.Unit)
 
         # Get user unit
```

Comment:
Other things it would be worth testing:
* If a user tries to `add_unit('store0_volt', 'millisecond')` or similar, do subsequent `get_unit` calls get very confused?
* If units named like 'my_unit' and 'also_my_unit' are added, does this work? Are both looked up correctly with `get_unit`?
* What about 'my_unit' and 'my_unit_again'?
* Or 'my_unit' and 'another_my_unit2'?

Reply:
Not sure I understand the need for the last 3?

Reply:
It's basically making sure the regex replacement doesn't do anything silly.